### ISSUE TITLE
Unable to list packages

### ISSUE THREAD

**Issue by babobski** [Komodo/KomodoEdit#3385]
### Short Summary
I'm currently unable to list my packages.

![afbeelding](https://user-images.githubusercontent.com/3530262/33232522-255cd7b0-d208-11e7-931f-d1adff0a453c.png)

I'm getting the following error:

```

[2017-11-25 17:38:46,861] [INFO] codeintel/process: CodeIntel started on port 50107
[2017-11-25 17:38:46,861] [DEBUG] codeintel/process: Calling callbacks for: started, number: 1
[2017-11-25 17:38:52,073] [WARNING] koLanguage: Asked for unknown language: 'Django XHTML Template'
[2017-11-25 17:38:52,073] [WARNING] koLanguage: Asked for unknown language: 'Django Text Template'
[2017-11-25 17:38:52,073] [WARNING] koLanguage: Asked for unknown language: 'Django Template'
[2017-11-25 17:38:52,073] [WARNING] koLanguage: Asked for unknown language: 'Laravel Blade Template'
[2017-11-25 17:38:52,073] [WARNING] koLanguage: Asked for unknown language: 'SGML'
[2017-11-25 17:38:52,089] [WARNING] koLanguage: Asked for unknown language: 'Django XML Template'
[2017-11-25 17:38:52,105] [WARNING] koLanguage: Asked for unknown language: 'Smarty Template'
[2017-11-25 17:38:52,105] [WARNING] koLanguage: Asked for unknown language: 'Django HTML Template'
[2017-11-25 17:38:52,121] [WARNING] koLanguage: Asked for unknown language: 'UDL'
[2017-11-25 17:39:46,075] [WARNING] console-logger: Key event not available on GTK2: key="u" modifiers="control,shift" (1) in chrome://komodo/content/komodo.xul:0
[2017-11-25 17:40:06,667] [ERROR] commando-scope-packages: Unable to retrieve package listing for addons
Traceback from ERROR in 'commando-scope-packages' logger:
    Logger.prototype.error@chrome://komodo/content/sdk/logging.js:411:27
    this._getAvailablePackagesByKind/<@chrome://scope-packages/content/packages.js:434:17
    this.request/req.onreadystatechange@chrome://komodo/content/sdk/ajax.js:75:17

[2017-11-25 17:40:06,799] [ERROR] commando-scope-packages: Unable to retrieve package listing for macros
Traceback from ERROR in 'commando-scope-packages' logger:
    Logger.prototype.error@chrome://komodo/content/sdk/logging.js:411:27
    this._getAvailablePackagesByKind/<@chrome://scope-packages/content/packages.js:434:17
    this.request/req.onreadystatechange@chrome://komodo/content/sdk/ajax.js:75:17

[2017-11-25 17:40:06,874] [ERROR] commando-scope-packages: Unable to retrieve package listing for toolbox
Traceback from ERROR in 'commando-scope-packages' logger:
    Logger.prototype.error@chrome://komodo/content/sdk/logging.js:411:27
    this._getAvailablePackagesByKind/<@chrome://scope-packages/content/packages.js:434:17
    this.request/req.onreadystatechange@chrome://komodo/content/sdk/ajax.js:75:17

[2017-11-25 17:40:06,937] [ERROR] commando-scope-packages: Unable to retrieve package listing for schemes
Traceback from ERROR in 'commando-scope-packages' logger:
    Logger.prototype.error@chrome://komodo/content/sdk/logging.js:411:27
    this._getAvailablePackagesByKind/<@chrome://scope-packages/content/packages.js:434:17
    this.request/req.onreadystatechange@chrome://komodo/content/sdk/ajax.js:75:17

[2017-11-25 17:40:07,005] [ERROR] commando-scope-packages: Unable to retrieve package listing for skins
Traceback from ERROR in 'commando-scope-packages' logger:
    Logger.prototype.error@chrome://komodo/content/sdk/logging.js:411:27
    this._getAvailablePackagesByKind/<@chrome://scope-packages/content/packages.js:434:17
    this.request/req.onreadystatechange@chrome://komodo/content/sdk/ajax.js:75:17

[2017-11-25 17:40:07,107] [ERROR] commando-scope-packages: Unable to retrieve package listing for languages
Traceback from ERROR in 'commando-scope-packages' logger:
    Logger.prototype.error@chrome://komodo/content/sdk/logging.js:411:27
    this._getAvailablePackagesByKind/<@chrome://scope-packages/content/packages.js:434:17
    this.request/req.onreadystatechange@chrome://komodo/content/sdk/ajax.js:75:17

[2017-11-25 17:40:07,269] [ERROR] commando-scope-packages: Unable to retrieve package listing for keybinds
Traceback from ERROR in 'commando-scope-packages' logger:
    Logger.prototype.error@chrome://komodo/content/sdk/logging.js:411:27
    this._getAvailablePackagesByKind/<@chrome://scope-packages/content/packages.js:434:17
    this.request/req.onreadystatechange@chrome://komodo/content/sdk/ajax.js:75:17

```

### Platform Information
Komodo IDE, version 11.0.1, build 90797, platform win32-x86

**Comment by Naatan** [comment]
You were likely seeing some down-time on our server. Does the issue still persist today?

**Comment by babobski** [comment]
@Naatan just tested it, but with the same results.
Also tested Komodo 10. but here the listing works fine.

**Comment by Naatan** [comment]
What about this new firewall of yours? ;)

**Comment by babobski** [comment]
I thought the same.
Things I tried:
- snoozing firewall
- snoozing antivirus
- kill the firewall and antivirus program
- Run komodo as administrator

All above options didn't work.
It also has a kind of application mananger.
Here I configured Komodo and python to be super trusted and allowed access to everyting.
But this had also no effect.

The last thing I tried is uninstalling the sucker ( ZoneAlarm ).
And that did the trick.

![afbeelding](https://user-images.githubusercontent.com/3530262/33365592-fcd981d8-d4e8-11e7-94b0-7423833ea1a1.png)
